rasa/server.py

```diff
@@ -1123,13 +1123,13 @@ async def evaluate_stories(
             "evaluate your model.",
         )
 
-        test_data = _test_data_file_from_payload(request, temporary_directory, ".md")
+        test_data = _test_data_file_from_payload(request, temporary_directory, ".yml")
```

Comment:
we can also dump the code [here](https://github.com/RasaHQ/rasa/blob/66a9e9a7adbf5c4b4abee3d11b2d3424dc4f2be1/rasa/server.py#L1470-L1476), no?